adt-pulse-mqtt/Dockerfile

```diff
@@ -9,10 +9,10 @@ ENV NODE_ENV production
 #RUN apk add --update nodejs=~14
 #RUN apk add --update nodejs-npm=~14
 
-# Install node and npm (Node 18 LTS)
```

Comment:
Technically, you are not guaranteed to be installing either Node 18 or LTS here.

Reply:
I just pulled your docker image and ran it locally.  Your assumption about the Balenalib images being "stuck" on an older version does not actually appear to be correct as far as I can tell.  The image is running Node 20.  You're pulling it via `apk`.
![image](https://github.com/user-attachments/assets/5bfa8bfc-2b0d-47fa-8f5d-d80715b6bc64)

Reply:
I think what you intended to do here is use the `balenalib/amd64-alpine-node:latest-3.18` image which would come pre-built with Node 18.  As would simply using `node/18-alpine` for that matter.  But what you're doing here is using a base image of `balenalib` and then assuming that running `apk` would somehow stick to Node 18.  It will pull the latest from the configured repositories and that is:
```
/etc/apk # cat repositories
https://dl-cdn.alpinelinux.org/alpine/v3.18/main
https://dl-cdn.alpinelinux.org/alpine/v3.18/community
```

Reply:
Ah, I must have gotten that image somehow which is why I made the edits I made.  Still, seems like it would be simpler/faster to use the pre-built `balenalib/amd64-alpine-node:latest-3.18` wouldn't it?

Reply:
Boy, I gotta say.  The `balenalib` stuff is a bit of a mess.  I don't understand it.  Look at their pre-built `balenalib/amd64-alpine-node:latest-3.18` image [docker manifest](https://hub.docker.com/layers/balenalib/amd64-alpine-node/latest-3.18/images/sha256-b6ac83ba2172fd594e097af948ec6c123f41a971113d3895e96ebaffd5f61c17?context=explore):
```
ENV NODE_VERSION=19.6.1
```
WAT?

Reply:
Given that the `-node` images don't seem to make any sense with their own documentation or patterns, I don't blame you. Just back away quietly and hope it keeps working. 😆